qlf_fasm/qlf_fasm.py

```diff
@@ -326,16 +327,20 @@ def disassemble_bitstream(self, bitstream, emit_unset=False):
         Disassembles a bistream.
         """
         features = []
+        force_bit_features = []
 
         def emit_feature(feature):
+            force_str = "force {}=1'b{};".format(feature, int(value))
+            str = feature
```

Comment:
Please change `str` to a different name. This one shadows the built-in Python type and may cause issues with debugging.

Reply:
Fixed this in the latest checkin

Reply:
> I'm confused about how `force_bit_features` differ from `features` in case when you enable emission of all set and unset features. If it carries the same information then why add it?

force bit file format is like (assuming only forcing set bits):
force <signal_name>=1'b1; 

Also, we need to ensure that preferably we dump all these signals in scalar form. Please let me know if you are already handling it and creating such file.